core/src/main/codegen/templates/Parser.jj

```diff
@@ -324,7 +324,11 @@ void SqlSelectKeywords(List<SqlLiteral> keywords) :
 void SqlInsertKeywords(List<SqlLiteral> keywords) :
 {}
 {
-    E()
+    // I looked through the various available parsers, none of them seemed to extend this in any way,
```

Comment:
Yea so this probably confirms that I was wrong about the being able to overwrite things. You can remove this comment since it won't be helpful for future people reading the code.

I've asked on the mailing list about this. It feels like these must be here for some reason that I just don't know how to hook into properly and I want to figure out if we're doing something wrong or if I'm just making a faulty assumption.

Reply:
Done